synapse/push/mailer.py

```diff
@@ -97,7 +97,7 @@
     "img": ["src"],
 }
 # When bleach release a version with this option, we can specify schemes
-# ALLOWED_SCHEMES = ["http", "https", "ftp", "mailto"]
```

Comment:
It looks like we still have a minimum version of `bleach>=1.4.3` in `pyproject.toml`. We would need to update that to be `bleach>=1.5` before making this change. 1.5 is ancient (from Nov 2016) so I think it would be OK to do that too.

Reply:
We should double-check with the packagers too.